src/braket/circuits/angled_gate.py

```diff
@@ -41,22 +50,51 @@ def __init__(self, angle: float, qubit_count: Optional[int], ascii_symbols: Sequ
         super().__init__(qubit_count=qubit_count, ascii_symbols=ascii_symbols)
         if angle is None:
             raise ValueError("angle must not be None")
-        self._angle = float(angle)  # explicit casting in case angle is e.g. np.float32
+        if issubclass(type(angle), FreeParameterExpression):
```

Comment:
why not isinstance instead?

Reply:
Good catch; why wasn't this
```suggestion
        if isinstance(angle, FreeParameterExpression):
```
from the beginning? Seems more intuitive to write.